EIPS/eip-2935.md

```diff
@@ -66,9 +59,11 @@ def resolve_blockhash(block: Block, state: State, arg: uint64):
   return state.load_slot(HISTORY_STORAGE_ADDRESS, arg % HISTORY_SERVE_WINDOW)
 ```
 
+<!-- TODO: Do we need to update the implementation because there is not history backfilling? -->
```

Comment:
no, because this is only really a problem one verkle activates (presumably people will keep using `BLOCKHASH` until the buffer is filled enough that it becomes interesting to use the contract. I would, however, add a note to explain just what i said.

Reply:
also blockhash only serves its original 256 window right?